sabnzbd/downloader.py

```diff
@@ -95,7 +95,7 @@ def __init__(
 
         self.busy_threads: List[NewsWrapper] = []
         self.idle_threads: List[NewsWrapper] = []
-        self.next_article_search: float = 0
+        self.next_article_search: float = 0.0
```

Comment:
There is no performance difference between 0 and 0.0.

```$ python -m timeit 'next_article_search: float = 0'
20000000 loops, best of 5: 11.1 nsec per loop

$ python -m timeit 'next_article_search: float = 0.0'
20000000 loops, best of 5: 11.1 nsec per loop```

Reply:
I know, it just looked not right to me to define it float but assign an integer.